charts/kyverno/Chart.yaml

```diff
@@ -9,15 +9,22 @@ keywords:
   - kubernetes
   - nirmata
   - policy agent
+  - policy
   - validating webhook
-  - admissions controller
+  - admission controller
+  - mutation
+  - mutate
+  - validate
+  - generate
+  - supply chain
+  - security
 home: https://kyverno.io/
 sources:
   - https://github.com/kyverno/kyverno
 maintainers:
-  - name: Nirmata
-    url: https://kyverno.io/
-kubeVersion: ">=1.16.0-0"
+  - name: Kyverno Maintainers
+    email: cncf-kyverno-maintainers@lists.cncf.io
+kubeVersion: ">=1.25.0-0"
```

Comment:
Why `1.25`? What changed between `1.10.x` and `1.11.x` to drop compatibility with `1.24` as per the [compatibility matrix](https://kyverno.io/docs/installation/#compatibility-matrix)?